frontend/src/cljs/imas_seamap/map/events.cljs

```diff
@@ -5,7 +5,7 @@
   (:require [clojure.string :as string]
             [re-frame.core :as re-frame]
             [cljs.spec.alpha :as s]
-            [imas-seamap.utils :refer [encode-state ids->layers first-where index-of]]
+            [imas-seamap.utils :refer [encode-state ids->layers first-where index-of append-query-params-from-map]]
```

Comment:
We've possibly had this discussion, but that `append-query-params-from-map` could probably be simplified to `append-query-params` (most interfaces along these lines use maps/dicts, and there's no real value in specifying the type)

Reply:
Yep, absolutely right, fixed